src/scss/_brand.scss

```diff
@@ -43,17 +38,15 @@ $_o-labels-shared-brand-config: (
 			),
 			'lifecycle-beta': (
 				background-color: oColorsGetPaletteColor('paper'),
-				border-color: oColorsMix('black', 'paper', 20),
-				text-color: oColorsMix('black', 'paper', 80),
+				border-color: oColorsGetPaletteColor('black-20'),
+				text-color: oColorsGetPaletteColor('black-80'),
 				padding-vertical: oTypographySpacingSize(1),
```

Comment:
`padding-vertical`, `padding-horizontal`, `text-transform` can be removed from `lifecycle-beta` brand config now?